src/appEntry.scss

```diff
@@ -27,11 +27,6 @@
     margin-left: 10pt;
```

Comment:
A [naming convention](http://thesassway.com/advanced/modular-css-naming-conventions) is still missing here. You mixed **dash-case** and **camelCase**